server/spec/models/user_spec.rb

```diff
@@ -1,5 +1,54 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  subject{
+    User.new(
+      usertype: 'company representative',
+      username: 'user1',
+      email: 'user1@tamu.edu',
+      password: 'password',
+      password_confirmation: 'password'
+    )
+  }
+
+  it 'is valid with valid attributes' do
+    expect(subject).to be_valid
+  end
+
+  it 'is not valid if not a valid usertype' do
+    usertypes = ['company representative', 'student', 'faculty', 'admin', 'volunteer']
+    if usertypes.include?(subject.usertype) == false
```

Comment:
Since your `subject` variable will always remain the same (and valid), non of your if-clauses will be executed, which defeats the purpose of the tests

Reply:
see this https://www.youtube.com/watch?v=fSTjQsWrfF4, it says otherwise

Reply:
And most of the RSpec tests overlap with the tests in the #6. So you only need to fix the `is not valid if not a valid usertype` only that one. Other tests are mostly covered.

Reply:
So please first focus on the cucumber testing